doc/source/locality.rst

```diff
@@ -7,11 +7,16 @@ Locality Module
 .. autosummary::
     :nosignatures:
 
+    freud.locality.AABBQuery
+    freud.locality.LinkCell
     freud.locality.NeighborList
     freud.locality.NeighborQuery
     freud.locality.NeighborQueryResult
-    freud.locality.AABBQuery
-    freud.locality.LinkCell
+    freud.locality.PairCompute
+    freud.locality.RawPoints
+    freud.locality.SpatialHistogram
+    freud.locality.SpatialHistogram1D
```

Comment:
We probably should hide about half of these, but we can do that in a separate PR that properly renames them everywhere.

Reply:
I created #512.